packages/icons/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+- Update `check` to be more optically balanced. ([#72101](https://github.com/WordPress/gutenberg/pull/72101))
+- Non-breaking, but major change: Switch to SVG icons as canonical source format, letting the system auto-generate the React elements and index. ([#71878](https://github.com/WordPress/gutenberg/pull/71878))
```

Comment:
@mcsf I had some headaches rebasing this branch, and my trunk was in a bit of a messy state, and I'm seeing this line as green now. Did I rebase incorrectly? I believe you were involved with this piece. Thanks for looking.

Reply:
Thank you, should be addressed now!

Reply:
Thanks for the quick response, @t-hamano!